src/vs/workbench/parts/search/browser/searchResultsView.ts

```diff
@@ -103,6 +106,14 @@ export class SearchDataSource implements IDataSource {
 		if (numChildren <= 0) {
 			return false;
 		}
+		const fileMatchCount = tree.getHTMLElement().children.length;
```

Comment:
This should not depend on the DOM.

Reply:
Yeah, it will expand automatically if just 1 file matched. I'll just remove it.
Thanks.
https://github.com/Microsoft/vscode/blob/d2148923f578ed65cec6c8a10f396933b3d1d054/src/vs/workbench/parts/search/browser/searchView.ts#L1166-L1178